build/gulpfile.vscode.web.js

```diff
@@ -110,6 +115,8 @@ const vscodeWebEntryPoints = !isAMD() ? [
   buildfile.workerBackgroundTokenization,
   buildfile.keyboardMaps,
   buildfile.workbenchWeb(),
+  // Gitpod Integration
+  buildfile.codeWeb,
```

Comment:
why is this needed isn't it already included in `buildfile.workbenchWeb()` ?

Reply:
There's no `workbench {}.js` without this line